pkg/build.sh

```diff
@@ -38,10 +38,12 @@ fpm \
 	--before-remove before-remove.sh \
 	--after-remove after-remove.sh \
 	--architecture amd64 \
-	--config-files opt/wavefront/wavefront-proxy/conf \
+	--config-files etc/wavefront/wavefront-proxy \
```

Comment:
everything in this directory will be a sticky conf file -- is that what you want? (i think earlier we talked about keeping a .default config file somewhere)